packages/internal/src/__tests__/fixtures/nestedPages/web/src/Routes.js

```diff
@@ -0,0 +1,59 @@
+// In this file, all Page components from 'src/pages` are auto-imported. Nested
+// directories are supported, and should be uppercase. Each subdirectory will be
+// prepended onto the component name.
+//
+// Examples:
+//
+// 'src/pages/HomePage/HomePage.js'         -> HomePage
+// 'src/pages/Admin/BooksPage/BooksPage.js' -> AdminBooksPage
+
+import { Private, Route, Router, Set } from '@redwoodjs/router'
+
+import AdminLayout from 'src/layouts/AdminLayout/AdminLayout'
+import MainLayout from 'src/layouts/MainLayout/MainLayout'
+
+import JobsPage from 'src/pages/Jobs/JobsPage'
+import JobPage from 'src/pages/Jobs/JobPage'
```

Comment:
```suggestion
import JobPage, { someRandomExport } from 'src/pages/Jobs/JobPage'
```

Would love to see a test for combined default+named imports like this, to make sure it doesn't throw the logic in the babel plugin off. I.e. make sure there's still no loader generated.

Reply:
Done!